src/sentry/static/sentry/app/views/settings/components/dataScrubbing/index.tsx

```diff
@@ -115,161 +93,73 @@ class DataScrubbing<T extends ProjectId = undefined> extends React.Component<
     }
   }
 
-  loadSourceSuggestions = async () => {
-    const {organization, projectId} = this.props;
-    const {eventId} = this.state;
-
-    if (!eventId.value) {
-      this.setState(prevState => ({
-        sourceSuggestions: valueSuggestions,
-        eventId: {
-          ...prevState.eventId,
-          status: undefined,
-        },
-      }));
-      return;
+  successfullySaved(
+    response: T extends undefined ? Organization : Project,
+    successMessage: string
+  ) {
+    const {onSubmitSuccess} = this.props;
+    this.setState({rules: convertRelayPiiConfig(response.relayPiiConfig)});
+    addSuccessMessage(successMessage);
+    if (onSubmitSuccess) {
+      onSubmitSuccess(response);
```

Comment:
You could write this as `onSubmitSuccess?.(response)`.

Not sure if we want to start promoting that, but I have been.